### ISSUE TITLE
NTLite dependancy

### ISSUE THREAD

**Issue by he3als** [he3als/gooseISO#8]
NTLite can be removed as a dependency in the future, there's not **too much** things that is done in NTLite.

**Comment by amitxv** [comment]
DISM all the way

**Comment by he3als** [comment]
Yeah, sadly that means that there probably won't be any Linux support. Maybe I could use WimLib, but I doubt it? https://wimlib.net/

**Comment by Zusier** [comment]
you can, wimlib is cross platform

**Comment by he3als** [comment]
> you can, wimlib is cross platform

Would it work with integrating drivers (as an example) though?

**Comment by Zusier** [comment]
No, wimlib is for exporting WIMs its not a full dism port as that isnt the purpose of it

**Comment by he3als** [comment]
DISM is probably going to have to be used then.
